src/features/events/l10n/messageIds.ts

```diff
@@ -54,6 +54,8 @@ export default makeMessages('feat.events', {
     move: m('Move'),
     moveInstructions: m('Drag the pin to select location.'),
     noDescription: m('No description'),
+    noRelatedEvents: m('There are no other events in this location.'),
```

Comment:
I think we should clarify here that we mean _parallel_ events, i.e. events that happen at the same time as the event we're editing. How could we do that?

Reply:
"There are no other events here at the same time." maybe?